client/state/analytics/reducer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+
+export const isTracking = ( state = false, { type } ) => {
```

Comment:
> I'd suggest to name this one tracking, and export default combineReducers( { tracking } );.

@ockham that feels like it goes against our naming conventions and it's certainly not my preference. don't we like to indicate boolean values with something in the name? if it's `tracking` for example it could also connote things like a list of ids we are tracking

in any case, I think `isSomething` is the de-facto standard in our other reducers, no?